Dockerfile.upstream

```diff
@@ -11,14 +11,14 @@ FROM fedora:38
 ARG OC_VERSION=latest
 ARG UMOCI_VERSION=latest
 
-RUN curl -LJo "openshift-client-linux-${OC_VERSION}.tar.gz" "https://mirror.openshift.com/pub/openshift-v4/amd64/clients/ocp/${OC_VERSION}/openshift-client-linux.tar.gz" \
+RUN dnf -y update && dnf install -y wget binutils go file runc podman jq skopeo && dnf clean all
+RUN wget -O "openshift-client-linux-${OC_VERSION}.tar.gz" "https://mirror.openshift.com/pub/openshift-v4/amd64/clients/ocp/${OC_VERSION}/openshift-client-linux.tar.gz" \
```

Comment:
What is the motivation behind the curl -> wget change? Might make sense to add it to commit description.

Reply:
It's a copy/paste from `Dockerfile`, which I was taking as a kind of source of truth for the fix/sync:
https://github.com/openshift/check-payload/blob/09a33418ded3f9217f8e511076ded4613f04ec59/Dockerfile#L15-L16

Reply:
Makes sense